src/js/post-excerpt/panel.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
```

Comment:
@dkotter, this is good, but I always prefer to use `import` packages when possible because this gives better code completion in the editor.

Reply:
While I tend to agree, I set this up to match what we're doing in other JS files in this project. I like to try and keep things consistent but open to feedback if you think we should change this here.